src/network/dandelion.py

```diff
@@ -177,21 +195,19 @@ def expire(self):
                 invQueue.put(row)
         return toDelete
 
-    def reRandomiseStems(self):
+    def reRandomiseStems(self, pool):
```

Comment:
We could use lazy initialisation on `pool` too and keep it as an attribute, then we don't need to pass it later.

Reply:
Done